scripts/run_integration_tests_aarch64.sh

```diff
@@ -11,15 +11,15 @@ build_spdk_nvme() {
     SPDK_DIR="$WORKLOADS_DIR/spdk"
     SPDK_REPO="https://github.com/spdk/spdk.git"
     SPDK_DEPLOY_DIR="/usr/local/bin/spdk-nvme"
-    checkout_repo "$SPDK_DIR" "$SPDK_REPO" master "6301f8915de32baed10dba1eebed556a6749211a"
+    checkout_repo "$SPDK_DIR" "$SPDK_REPO" master "ceefb46358f845d6c4e9f9e18d02ec20a7098ff7"
```

Comment:
The problem you are resolving is about DPDK, but why do you modify `SPDK`?

Reply:
There are some error when build on old version of spdk,I don't know how to fix it.
```
/root/workloads/spdk/libvfio-user/samples/client.c: In function 'fake_guest':
/root/workloads/spdk/libvfio-user/samples/client.c:831:9: error: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [-Werror=deprecated-declarations]
  831 |         MD5_Init(&md5_ctx);
      |         ^~~~~~~~
```